lnrpc/routerrpc/router_server.go

```diff
@@ -740,10 +744,55 @@ func (s *Server) trackPayment(identifier lntypes.Hash,
 	case err != nil:
 		return err
 	}
+
+	// Stream updates to the client
+	err = s.trackPaymentStream(
+		subscription,
```

Comment:
style nit: we try to make method calls a bit more compact by fitting as many arguments on one line as possible (while respecting the max line length).

Reply:
Same below.

Reply:
Done